charts/caraml/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.1.1
```

Comment:
Why increase the chart version when there are no changes made?  We should increment the version when we create the umbrella chart, right?

Reply:
Might have been incremented automatically, let me check

Reply:
https://github.com/caraml-dev/helm-charts/actions/runs/3051086023/jobs/4918900383#step:8:66 version needs to be bumped else linting fails

Reply:
Looks like there is a whitespace change in `charts/caraml/templates/NOTES.txt`, hence it's asking you for a version bump. Could you checkout the file and make sure there is no diff under CaraML chart, That should fix your lint error.

Reply:
notes change is from the eof fix. I think previously there was no newline at the end of the file, so this change was added after passing thru the eof fixer. As discussed, will leave it first.

Reply:
Should be fine with this change. resolving the conversation